README.md

```diff
@@ -28,18 +33,19 @@ You will still need to run the following too...
     sudo apt-get install git zlib1g-dev nodejs
     sudo gem install bundler
 
-### Checkout Code and Run Jekyll
+#### Check Out Code and Run Jekyll
     git clone git@github.com:<user>/ev3dev.github.io
     cd ev3dev.github.io
     bundle install
     bundle exec jekyll serve --safe --incremental &
     www-browser http://localhost:4000 &
     # work, work, work
 
-Installing Jekyll in  Windows
----
+### Installing Jekyll on  Windows
 
-- Follow the instructions in step 1 [here](http://jekyll-windows.juthilo.com/1-ruby-and-devkit/) (Installing Ruby and the Ruby DevKit).
+Firstly, follow the instructions in step 1
```

Comment:
"Firstly" sounds too formal to me. "First" is just my personal preference (and consistent with the rest of the website).